storage/errors.go

```diff
@@ -78,3 +78,6 @@ var ErrInvalidNumberOfEpochsToSave = errors.New("invalid number of epochs to sav
 
 // ErrInvalidNumberOfActivePersisters signals that an invalid number of active persisters has been provided
 var ErrInvalidNumberOfActivePersisters = errors.New("invalid number of active persisters")
+
+//ErrClosingPersisters signals that not all persisters was closed
```

Comment:
were closed *
Also, I don't think it is enough. We won't know why any persister failed to close. So, at least log the errors when a persister.Close() fails.